plugins/ltac2/tac2tactics.ml

```diff
@@ -456,3 +456,21 @@ let inversion knd arg pat ids =
 let contradiction c =
   let c = Option.map mk_with_bindings c in
   Contradiction.contradiction c
+
+let congruence n l = 
+  let n = match n with
+  | None -> 1000
```

Comment:
Don't hardcode the values, simply expose this function argument as an integer. Same remarks for the variant below and the list arguments.

Reply:
That's also fine.

Reply:
changed it so that the defaults are in cctac.ml and g_congruence.mlg just calls it wit options so the ltac2 version also calls it with options